docs/content/users/install/docker-installation.md

```diff
@@ -49,6 +49,8 @@ You’ll need a Docker provider on your system before you can [install DDEV](dde
 
     Docker Desktop for Mac can be installed via Homebrew (`brew install homebrew/cask/docker`) or can be downloaded from [docker.com](https://www.docker.com/products/docker-desktop). It has long been supported by DDEV and has extensive automated testing.
 
+    If you use Docker Desktop for Mac, the `VirtioFS` option, while performant, is *not* recommended at this time and can cause many mysterious problems. Also,, when Mutagen is enabled, the performance implications of `VirtioFS` are not significant.
```

Comment:
Ah, I missed this while I was reviewing the other bits—sorry! 

I’d activate the language here for warmth and [tonal consistency](https://ddev.readthedocs.io/en/stable/developers/writing-style-guide/#use-active-third-person), add a link, and fix the double comma. 

I think it whittles down pretty well:

```
We do not recommend the “VirtioFS” option with Docker Desktop for Mac. While it’s performant, it can cause mysterious problems that are not present with [Mutagen](performance.md#mutagen)—which offers comparable performance when enabled.
```